azurerm/internal/services/web/function_app.go

```diff
@@ -131,6 +131,26 @@ func schemaAppServiceFunctionAppSiteConfig() *pluginsdk.Schema {
 					Optional:     true,
 					ValidateFunc: validation.StringInSlice([]string{"1.8", "11"}, false),
 				},
+
+				"minimum_elastic_instance_count": {
```

Comment:
for consistency with the tense used in the provider
```suggestion
				"elastic_instance_minimum": {
```

Reply:
I am concerned that the naming conventions are different from the `pre_warmed_instance_count` and `maximum_elastic_worker_count` that already exist, does this mean that the provider as a whole will be moving to the new conventions?

Reply:
in the long term that is the intention - you can just toss a todo comment on those properties for us to tidy up when we start on 3.0

Reply:
Thank you very much. I could see that this is a change for 3.0. I'll rename this property!